docs/Learning-Environment-Examples.md

```diff
@@ -254,6 +254,29 @@ return.
 * Benchmark Mean Reward for `CrawlerStaticTarget`: 2000
 * Benchmark Mean Reward for `CrawlerDynamicTarget`: 400
 
+## Worm
+
+![Worm](images/worm.png)
+
+* Set-up: A worm with a head and 3 body segments.
+* Goal: The agents must move its body toward the goal direction.
+  * `WormStaticTarget` - Goal direction is always forward.
+  * `WormDynamicTarget`- Goal direction is randomized.
+* Agents: The environment contains 10 agents with same Behavior Parameters.
+* Agent Reward Function (independent):
+  * +0.01 times body velocity in the goal direction.
+  * +0.01 times body direction alignment with goal direction.
+* Behavior Parameters:
+  * Vector Observation space: 57 variables corresponding to position, rotation,
+    velocity, and angular velocities of each limb plus the acceleration and
+    angular acceleration of the body.
+  * Vector Action space: (Continuous) Size of 9, corresponding to target
+    rotations for joints.
+  * Visual Observations: None
+* Float Properties: None
+* Benchmark Mean Reward for `WormStaticTarget`: 150
```

Comment:
This doesn't seem right, since for Crawler, the static reward is 5x the dynamic reward.

Reply:
i retrained to confirm benchmark. it scored 200 so i updated the doc. several reasons there is a huge difference.
1. I increased the gravity to 1.5x so the worm is experiencing more gravity. The crawler can go much faster because it's essentially floating
2. The worm has much weaker joints than the crawler. If the joints are strong it doesn't have natural oscillating motion. weaker joints mean it can't rely on launching itself foward like crawler does.
3. all body segments on the worm are of mass 10. this is to give it a heavier look. 

overall these changes were made to give the worm a less floaty look. I expected the benchmark to be lower.

Reply:
Looks like you updated worm dynamic to be higher than worm static. Is that right? I know the overall will be lower for both, but shouldn't static be better than dynamic?

Reply:
yes. error on my part. i updated it.